sbi/inference/posteriors/direct_posterior.py

```diff
@@ -176,6 +177,10 @@ def log_prob(
             track_gradients: Whether the returned tensor supports tracking gradients.
                 This can be helpful for e.g. sensitivity analysis, but increases memory
                 consumption.
+            leakage_correction_params: A `dict` of keyword arguments to override the
+                default values of `leakage_correction()`. Possible options are:
+                `num_rejection_samples`, `force_update`, `show_progress_bars`, and
+                `rejection_sampling_batch_size`.
```

Comment:
Could you add 

Only used if `norm_posterior=True`.

?

Reply:
Sure, you mean in the doc string, right?

Reply:
yes, exactly. Thanks!

Reply:
Done. Let me know, if there is sth else.